src/app/beer_garden/events/processors.py

```diff
@@ -125,16 +128,47 @@ def register(self, processor, manage: bool = True):
 class HttpEventProcessor(QueueListener):
```

Comment:
I think this should be renamed and moved at this point... it's a lot more than a simple processor. More like a "HttpParentUpdater".

You may want to go ahead and think of the best way to organize the class structure here to separate "parent updater" stuff from "http" stuff. That'll be important to support other transports in the near future.